estate/models/estate_property.py

```diff
@@ -0,0 +1,42 @@
+from odoo import fields, models
+from datetime import timedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate property"
+
+    name = fields.Char('Name', required=True, default='My new house')
+    description = fields.Text('Description')
+    active = fields.Boolean(default=True)
+    postcode = fields.Char('Postcode')
+    date_availability = fields.Date('Available From',default=lambda self: fields.Date.today() + timedelta(days=90), copy=False)
```

Comment:
```suggestion
    date_availability = fields.Date('Available From',default=lambda self: fields.Date.context_today(self) + timedelta(days=90), copy=False)
```
Here, it's better to use `fields.Date.context_today()` for setting a default to the current date, because it correctly handles the user's timezones.